src/main/kotlin/org/jitsi/jibri/webhooks/v1/README.md

```diff
@@ -0,0 +1,46 @@
+# Jibri 'Webhooks'
+
+Jibri supports being configured with a list of base URLs on which it will hit certain endpoints with data.  These are
+not "true" webhooks as the endpoints are hard-coded, instead Jibri defines a "contract" which it expect a subscriber
+to implement at the given base URL.  Information about this contract is below.
+
+### Status updates
+Jibri pushes status updates consisting of its "busy status" (whether it is busy or idle) and its health.
```

Comment:
How often does it push these and is that configurable? I think the answer is every 60 seconds and on any state change but it helps to call it out in this readme.